packages/shared/src/messages/MessageProxy.spec.ts

```diff
@@ -0,0 +1,112 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { test, expect, beforeAll } from 'vitest';
+import { RpcBrowser, RpcExtension } from './MessageProxy';
+import type { Webview } from '@podman-desktop/api';
+
+let webview: Webview;
+let window: Window;
+let api: PodmanDesktopApi;
+
+beforeAll(() => {
```

Comment:
beforeEach might be better for test isolation

Reply:
I know :( but this is directly copied and pasted from ai-studio, should I push the "fix" upstream?